.github/workflows/publish_release.yaml

```diff
@@ -44,7 +37,7 @@ jobs:
           DEB_VER=$(cut -d'-' -f2 <<< $PKG_VER)
```

Comment:
Afaict we don't need the $DEB_VER var anymore, right? If so could remove it

Reply:
removed.